predicates/forwarded/forwarded.go

```diff
@@ -133,8 +133,18 @@ func parseForwarded(fh string) *forwarded {
 
 	f := &forwarded{}
 
-	for _, forwardedFull := range strings.Split(fh, ",") {
-		for _, forwardedPair := range strings.Split(strings.TrimSpace(forwardedFull), ";") {
+	// TODO: use range over func added in go1.23 like:
+	// for forwardedFull := range split(fh, ",") {
+	// 	for forwardedPair := range split(strings.TrimSpace(forwardedFull), ";") {
+	// 		// ...
+	// 	}
+	// }
+	// once go1.24 is released
```

Comment:
Actually it was released yesterday so we may bump min version https://github.com/zalando/skipper/pull/3406 and use range over func.

Reply:
Updated.